pkg/log/audit_test.go

```diff
@@ -0,0 +1,44 @@
+package log
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFormatComponentStatus(t *testing.T) {
+	// Tests
+	tests := []struct {
+		testName  string
+		component string
+		status    string
+		expected  string
+	}{
+		{
+			testName:  "Success test",
```

Comment:
Nit: Since these are all tests we don't need to include the word "test" in the names. Something cleaner could be "Successful component" for example.

Reply:
Ahhh, good to know. I'm still not familiar enough with the table testing output to have picked this up on my own, so thanks for pointing it out.